### DISCUSSION TITLE
Laravel Octane

### DISCUSSION THREAD

**Discussion by andrebreia** [statamic/cms#4446] [Troubleshooting] [answered]
We're trying to get Octane working on a Laravel app that also uses Statamic as the main site, but **sometimes** we get the error below:

<img width="1270" alt="Screenshot 2021-10-11 at 15 48 21" src="https://user-images.githubusercontent.com/17434432/136801274-c043c509-a922-4f91-8c56-c20dfff2c0c8.png">

Not sure if anyone ran into this issue? Or if Octane is simply not supported by Statamic?

Thanks in advance!

**Comment by jasonvarga** [comment, marked as answer]
Octane is not supported at the moment.

You may be able to get away with it in some cases, but clearly not everywhere.

**Reply by andrebreia** [reply]
Thanks, Jason. We'll leave it for now, and come back later to it if we can figure it out.

**Comment by fpolli** [comment]
Is Statamic still not compatible with Octane?

**Reply by jasonvarga** [reply]
It is still not compatible.

**Comment by macaws** [comment]
Wondering what it would take to make Octane work fully in Statamic? Seems that Octane and FrankenPHP have become a lot more popular - and for good reason. For those of us who use Eloquent Driver and not flat files, we could get some pretty crazy gains from having Octane. I've personally been using it on every single non-Statamic project because the difference in performance is night and day.

Happy to help with these efforts as well - but only if the Statamic team decides they want to support it.

**Reply by jimblue** [reply]
Maybe with v6... 🤞🏼

**Reply by edalzell** [reply]
V6 ain’t got nuthin to do with it. It’s the files
